packages/config/config/devices/0x0249/psr07.json

```diff
@@ -0,0 +1,85 @@
+// WeBeHome CPH PSR07
+// Smart Color Button
+{
+	"manufacturer": "WeBeHome",
+	"manufacturerId": "0x0249",
+	"label": "PSR07",
+	"description": "Smart Color Button",
+	"devices": [
+		{
+			"productType": "0x0002",
+			"productId": "0x0001",
+			"zwaveAllianceId": [2029, 2049]
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Basic Set Off Level",
+			"description": "",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Basic Set On Level",
+			"description": "",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"10": {
+			"label": "Auto Report Battery Time",
+			"description": "",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 127,
+			"defaultValue": 12,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"25": {
+			"label": "Percentage Set Method",
```

Comment:
The options don't make sense for this label IMO. Maybe there's a better one?

Reply:
I agree, but it's a device I've never seen before.  I changed this to "Dimmer Set Method" which seems to be more accurate per the manual.